src/app/App.js

```diff
@@ -70,15 +71,16 @@ export const App = ({ options }) => {
     withClasses: 'mainContainer__answersWrapper',
   });
 
-
-
-
   odpTrue.changeText('100% prawda');
```

Comment:
tutaj cos sie popsulo, bo tych propertiesow nie ma. Tych odpTrue itp.

Reply:
no to jest na developie... nie usuwam, bo nie moje i nie wiem czy się przyda komuś czy nie :-P

Reply:
Jak dla mnie dobrze usunac, bo to w ogole nie dziala :P Nie ma takich zmiennych jak odpTrue np. I wtedy nic się nie wykona po tym kodzie.